### DISCUSSION TITLE
Creador de widget feed para Blogger

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#18] [Announcements]
# Creador de widget feed para Blogger

Plugin que crea un widget del feed de Blogger, que genera código para insertar las últimas publicaciones en otro blog

https://www.zkreations.com/2023/02/creador-de-widget-feed-para-blogger.html

**Comment by TakoMics** [comment]
Muchísimas gracias por el aporte Daniel. Se me ocurren varios usos para este widget.

**Reply by danieIabel** [reply]
Espero que te sea de utilidad amigo, un saludo!
